modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_sidebar/CriteriaSidebarItem.es.js

```diff
@@ -0,0 +1,94 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import ClayIcon from '../shared/ClayIcon.es';
+import {DragSource as dragSource} from 'react-dnd';
+import {DragTypes} from '../../utils/drag-types.es';
+import getCN from 'classnames';
+
+/**
+ * Passes the required values to the drop target.
+ * This method must be called `beginDrag`.
+ * @param {Object} props Component's current props
+ * @returns {Object} The props to be passed to the drop target.
+ */
+function beginDrag({name}) {
+	return {
+		criterion: {
+			propertyName: name
+		}
+	};
+}
+
+class CriteriaSidebarItem extends Component {
+	static propTypes = {
+		connectDragSource: PropTypes.func,
+		dragging: PropTypes.bool,
+		label: PropTypes.string,
+		name: PropTypes.string,
+		type: PropTypes.string
+	};
+
+	_getIcon(type) {
+		let returnValue;
+
+		switch (type) {
+		case 'boolean':
```

Comment:
indentation looks off for this block. Unless its csf... 🤷‍♂️

Reply:
Yup...csf didn't like it indented 🤷‍♂️